### PR TITLE
Add CefVisitWebPluginInfo Wrapper

### PR DESCRIPTION
**Pull request by amaitland**
Implement `CefVisitWebPluginInfo` using a simple `TaskCompletionSource` to notify when all plugins have been visited.

Also added `custom://cefsharp/plugins` that outputs a basic html table showing plugin details.

### CODE REVIEW

File: CefSharp.Core/Cef.h [hunk 1]
```diff
@@ -454,6 +455,17 @@ namespace CefSharp
             return CefClearSchemeHandlerFactories();
         }
 
+        /// <summary>Visits all plugins using the provided Plugin Visitor.</summary>
+        /// <param name="visitor">A user-provided Plugin Visitor implementation.</param>
+        static Task<List<Plugin>^>^ GetPlugins()
```
Comment (jankurianski) [inline CefSharp.Core/Cef.h]: Visitor parameter comment should be removed.

Comment (amaitland) [inline CefSharp.Core/Cef.h]: Thanks :+1:

**Comment by amaitland** [comment]
@jankurianski I've updated the xml doc, any suggestions on the wording are most welcome :smile:

**Comment by jankurianski** [comment]
I think that's good.  I also checked out the PR to give it a spin and it works for me. :+1:

**Comment by amaitland** [comment]
> I also checked out the PR to give it a spin and it works for me.

Thanks :+1: Much appreciated :smile: Figure this might make debugging plugin problems a little easier, at least we can now see what's being detected.

**Pull request merged**